jenkins/vendor_images.yaml

```diff
@@ -1,3 +1,3 @@
 docker.io:
   images-by-tag-regex:
-    jenkins/jenkins: ^v?[0-9]+\.[0-9]+?\.[0-9]+$
```

Comment:
This regex isn't good. Please change it to `(?:^v?2\.(?:38[7-9]|39[0-9]|[4-9][0-9]{2}|[1-9][0-9]{3,})+?\.[0-9]+$)|(?:^v?(?:(?:[^0-2]{1})?(?:[3-9]{1}|[0-9]{2}))+\.[0-9]+?\.[0-9]+$)`. This will match anything >= v2.387.0.

Reply:
I've already pushed the change so you can remove your change to the image vendoring completely.